airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1975,6 +1975,27 @@ paths:
                 $ref: "#/components/schemas/WebBackendConnectionRead"
         "422":
           $ref: "#/components/responses/InvalidInputResponse"
+  /v1/web_backend/connections/updateNew:
```

Comment:
Given that this is the superseeded version of an already existing API, could we just name it `/v2/web_backend/connections/update`? Since we have a version number already in our API paths, this feels like the ideal use-case to leverage it :)